fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -1017,6 +1017,13 @@ identifierOrTextOrAsterisk
     | ASTERISK
     ;
 
+identifierOrTextOrParen
```

Comment:
seperate different hint into its own statement's syntax to let grammer more clear